crt_portal/cts_forms/forms.py

```diff
@@ -1009,7 +1011,9 @@ def __init__(self, *args, **kwargs):
         self.fields['assigned_section'] = ChoiceField(
             widget=ComplaintSelect(
                 label='Section',
-                attrs={'class': 'usa-select text-bold text-uppercase crt-dropdown__data'},
+                attrs={
+                    'class': 'usa-select text-bold text-uppercase crt-dropdown__data'
+                },
```

Comment:
Any reason `attrs` was automatically formatted? In general if this is not part of a cleanup then I'd leave this alone, as it adds to PR noise.